core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -255,8 +255,7 @@ export class ExplorationEditorPageComponent implements OnInit, OnDestroy {
           explorationData.param_changes));
       this.explorationAutomaticTextToSpeechService.init(
         explorationData.auto_tts_enabled);
-      this.explorationCorrectnessFeedbackService.init(
-        explorationData.correctness_feedback_enabled);
+      this.explorationCorrectnessFeedbackService.init(true);
```

Comment:
Why do we need this line (an the entire service) at all?

Reply:
removed other unnecessary services related to correctness feedback from the frontend, like PlayerCorrectnessFeedbackEnabledService and ExplorationCorrectnessFeedbackService, as they were only used for getting info about correctness feedback and responsible for changing it (which we don't need anymore).